src/store/wizardSlice.ts

```diff
@@ -656,6 +696,22 @@ export const wizardSlice = createSlice({
     setFirstBootScript: (state, action: PayloadAction<string>) => {
       state.firstBoot.script = action.payload;
     },
+    setUserName: (state, action: PayloadAction<string>) => {
+      state.userInfo.userName = action.payload;
+    },
+    setUserPassword: (state, action: PayloadAction<string>) => {
+      state.userInfo.password = action.payload;
+    },
+
+    setConfirmUserPassword: (state, action: PayloadAction<string>) => {
+      state.userInfo.confirmPassword = action.payload;
+    },
+    setUserSshKey: (state, action: PayloadAction<string>) => {
+      state.userInfo.sshKey = action.payload;
+    },
+    changeUserAdministrator: (state, action: PayloadAction<boolean>) => {
+      state.userInfo.administrator = action.payload;
+    },
```

Comment:
These would then be replaced by `addUser`, `editUser` and `removeUser`.

- `addUser` would dispatch a new entry, so it would just push a new object to `users` array.
- `removeUser` would filter through the `users` array and remove an item by probably a user name. For now it could also just rewrite the array with an empty one if we're working with just one user.
- `editUser` should be able to check whether the user is already there and update the values in the object. So it would probably try to find the correct user by name and then update existing values with new ones.

Reply:
`Administrator` adds the user to sudoers. It's just a boolean.